### PR TITLE
Create the typography>Fonts screens without functionality

### PR DESCRIPTION
**Pull request by carin91**
I added 2 new components WITHOUT functionality, with their storybooks.
Add to it, i configure the AppMenu to include children and some of its responsibilities were transferred to a new AppMenuGrid component

### CODE REVIEW

File: src/components/layout/AppMenu/index.tsx [hunk 1]
```diff
@@ -1,49 +1,29 @@
-import { AppMenuCard } from "@components/cards/AppMenuCard/index";
-import { Breadcrumbs, Grid, Stack, useMediaQuery } from "@inube/design-system";
+import { Breadcrumbs, Stack } from "@inube/design-system";
 import { PageTitle } from "../../PageTitle";
```
Comment (LinconOssa) [inline src/components/layout/AppMenu/index.tsx:2]: we can change this ../../PageTitle to simply @components/PageTitle

File: src/pages/people/outlets/fonts/form/RenderFormFonts/index.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
+import { fontMessagesConfig } from "../../config/fonts.config";
```
Comment (LinconOssa) [inline src/pages/people/outlets/fonts/form/RenderFormFonts/index.tsx:1]: The same can be done here with @pages?

**Review by LinconOssa — COMMENTED** [review]
I find these differences:
image
check that everything matches because according to the images I don't see it the same as figma

File: src/components/layout/AppMenu/index.tsx (hunk 1 above)
Comment (carin91) [reply src/components/layout/AppMenu/index.tsx:2]: Attended
image

File: src/pages/people/outlets/fonts/form/RenderFormFonts/index.tsx (hunk 2 above)
Comment (carin91) [reply src/pages/people/outlets/fonts/form/RenderFormFonts/index.tsx:1]: Attended
image

**Review by LinconOssa — APPROVED** [review]
meet the requirements

**Review by cmarin001 — APPROVED** [review]
meet the requirements

**Pull request merged**